libcontainer/system/sysconfig.go

```diff
@@ -10,3 +10,7 @@ import "C"
 func GetClockTicks() int {
 	return int(C.sysconf(C._SC_CLK_TCK))
 }
+
+func GetLongBit() int {
+	return int(C.sysconf(C._SC_LONG_BIT))
```

Comment:
Sorry, I meant to write that it should be conditional based on if `_SC_LONG_BIT` is set (not actually setting that value).